README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
+simulated devices are supported. You can use the **NI MAX** or **NI Hardware Configuration Utility**
+to verify and configure your devices.
```

Comment:
X Series isn't recommended - we support any DAQ device! Let's reword this a bit. (my formatting isn't good, sorry - not in an editor)

```suggestion
In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
system. Both physical and simulated devices are supported. The examples below use an X Series DAQ device (eg: PXIe-6363, PCIe-6363, or USB-6363). You can use the **NI MAX** or **NI Hardware Configuration Utility**
to verify and configure your devices.
```